### ISSUE TITLE
First-run message is incorrect in non-English languages

### ISSUE THREAD

**Issue by natemcmaster** [dotnet/cli#9379]
_From @Advanium on May 31, 2018 10:14_

The .NET Command Line tools in v2.1.300 display invalid commands in the ASP.NET Core section of the information that appears when new'ing up a solution.

```ps
dotnet new sln -n "Extensions"
```

This applies at least to the German localized version of the tooling:

![image](https://user-images.githubusercontent.com/515866/40776528-4a2085a0-64cb-11e8-9d2f-5d16b4effbbb.png)

Executing the command

```ps
dotnet tool install dotnet-dev-certs -g --version 2.1.0-preview1-final
```

results in a "format version missing" message, but eventually install the tool and display which version of it was installed. It also prints a usage message for how to use the dev-certs commandlet, which is also wrong:

![image](https://user-images.githubusercontent.com/515866/40776733-f3cf3ace-64cb-11e8-8922-906b839f5f15.png)

```ps
dotnet-dev-certs
```

This won't execute, because there's no tool `dotnet-dev-certs`. The first dash isn't supposed to be there, and the correct command would be:

```ps
dotnet dev-certs
```

I don't know if other output and/or localized content also contains invalid commands.
Can you verify this and patch it?

_Copied from original issue: aspnet/DotNetTools#445_

**Comment by natemcmaster** [comment]
@javiercn @livarcocc I thought we fixed this in https://github.com/dotnet/cli/pull/895. Do  we need to update another location?

**Comment by natemcmaster** [comment]
Looks like the translation wasn't completed. The English message is correct, but it's wrong in other languages. See https://github.com/dotnet/cli/tree/release/2.1.3xx/src/Microsoft.DotNet.Configurer/xlf.

You don't need to run `dotnet tool install` to acquire `dotnet dev-certs`. This command is bundled in the SDK. The message you are seeing is a left-over from preview1.

Moving this issue to the dotnet/cli repo.

**Comment by livarcocc** [comment]
Assigning to @nguerrera just to drive a round of loc insertions on the CLI/SDK.

**Comment by nguerrera** [comment]
This will be fixed by #9653
